app/session.go

```diff
@@ -113,6 +121,27 @@ func (a *App) RevokeAllSessions(userId string) *model.AppError {
 	return nil
 }
 
+// RevokeSessionsFromAllUsers will go through all the sessions active
+// in the server and revoke them
+func (a *App) RevokeSessionsFromAllUsers() *model.AppError {
```

Comment:
Could this be done directly in the store layer? I'm imagining a server with 100,000 users, for example, where there might be as much as 1,000,000 sessions. Will this approach of fetching all sessions from the store in one massive `GetAll()` query behave nicely with such a large number of rows in the session table?

Reply:
Easier that way, I'm afraid I didn't grasp the cluster part and thought I had to go one user at a time, so I had to retrieve all the sessions to ensure that. This is an easy fix which will end with less code even😁